classes/local/attributemapping.php

```diff
@@ -100,6 +100,10 @@ private function validate_table($table) {
      * @throws InvalidArgumentException If the field name is invalid
      */
     private function validate_field($table, $field) {
+        if (!$field) {
+            return;
+        }
```

Comment:
Allowing non-selected values as users could leave the field blank and save the form.

Reply:
This should be reverted once multiple selection is implemented.

Reply:
Yes, that's true as long as it's restricted to select only one of the fields and leave the other blank.